monitor/core/pom.xml

```diff
@@ -20,15 +20,181 @@
+					<groupId>org.restlet.jse</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>rsp-services-api</artifactId>
+					<groupId>polimi.deib</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>qos-models</artifactId>
+					<groupId>it.polimi.modaclouds.qos-models</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>knowledge-base-api</artifactId>
+					<groupId>it.polimi.modaclouds.monitoring.knowledge-base-api</groupId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<dependency>
+         	<groupId>monitoring-manager</groupId>
+         	<artifactId>monitoring-manager</artifactId>
+         	<scope>system</scope>
+         	<version>1.4</version>
+         	<systemPath>${basedir}/lib/monitoring-manager-1.4.jar</systemPath>
+      	</dependency>
+
+      	<dependency>
+      		<groupId>data-collector</groupId>
+			<artifactId>data-collector</artifactId>
+			<scope>system</scope>
+			<version>1.3-SNAPSHOT</version>
+			<systemPath>${basedir}/lib/data-collector-1.3-SNAPSHOT.jar</systemPath>
+		</dependency>
+	</dependencies>
+
+
+	<build>
+		<plugins>
+			<plugin>
+    			<artifactId>maven-antrun-plugin</artifactId>
+    			<executions>
+        			<execution>
+            			<id>download-files</id>
+            			<!--<phase>clean</phase>-->
+            			<phase>process-resources</phase>
+            			<goals>
+                			<goal>run</goal>
+            			</goals>
+            			<configuration>
+                			<target>
+                    			<!-- download files -->
+                    			<get src="http://archive.apache.org/dist/jena/binaries/jena-fuseki-1.1.1-distribution.zip"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>
+                    			<get src="https://magelan.googlecode.com/files/hyperic-sigar-1.6.4.zip"
+	        			            dest="${project.build.directory}"
+    	                		    verbose="false"
+        	                 		usetimestamp="true"/>
+                    			<get src="http://www.cs.uoi.gr/~dathanas/rsp-services-csparql-0.4.6.2-modaclouds-distribution.zip"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>
+                         		<!--<get src="http://www.cs.uoi.gr/~dathanas/monitoring-manager-1.4.jar"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>
+                         		<get src="https://github.com/imperial-modaclouds/modaclouds-data-collectors/releases/download/1.3-Snapshot/data-collector-1.3-SNAPSHOT.jar"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>-->
+                 			</target>
```

Comment:
These commented targets look like an alternative (not the best, but i'd be ok to merge with this at the moment) to having to add the jars to the repository. Is it commented for any reason @dionysiosAthanasopoulos ?